pulseguardian/guardian.py

```diff
@@ -263,62 +263,58 @@ def monitor_queues(self, queues, all_bindings):
             db_session.add(queue)
             db_session.commit()
 
-    def _exchange_from_queue(self, queue_data):
-        exchange = 'could not be determined'
-        detailed_data = pulse_management.queue(vhost=queue_data['vhost'],
-                                               queue=queue_data['name'])
-        if 'incoming' in detailed_data and detailed_data['incoming']:
-            exchange = detailed_data['incoming'][0]['exchange']['name']
-        return exchange
-
-    def warning_email(self, users, queue_data):
-        exchange = self._exchange_from_queue(queue_data)
-
+    def warning_email(self, users, queue_data, is_unbounded):
         subject = 'Pulse warning: queue "{0}" is overgrowing'.format(
             queue_data['name'])
-        body = '''Warning: your queue "{0}" on exchange "{1}" is
-overgrowing ({2} ready messages, {3} total messages).
-
-The queue will be automatically deleted when it exceeds {4} messages.
```

Comment:
I love how this reads if you don't see that as a format string. We'd be a very strict guardian.

Reply:
"This is a message bus, not a message sleeper car!"